src/platforms/go/common/troubleshooting.mdx

```diff
@@ -20,4 +20,4 @@ Since the Go SDK tries to read the source files from your local disk, it's possi
 
 This is common in Go because you can compile the binary, and then have no more need for the source code. Without access to the source code, however, we can’t map anything back, and we don’t support uploading Go source code with a release to stitch this data on.
 
-You can see [here](https://docs.sentry.io/platforms/go/serverless/#source-context) how to bunde the source code with the binary.
+You can see [in the Serverless section](serverless/#source-context) how to bundle the source code with the binary, which applies to other forms of deployment as well.
```

Comment:
Oh, the relative link here doesn't work for the guides. Probably for now we go with 

```suggestion
You can see [in the Serverless section](/platforms/go/serverless/#source-context) how to bundle the source code with the binary, which applies to other forms of deployment as well.
```

which will work for previews and prod, and from a guide will link out of the guide and into the Serverless section.

This really needs some re-organizing :)